Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from CoolProp.CoolProp import PropsSI


T_amb = 20  # Â°C
p_amb = 1.01325  # bar
fluid = "air"
dot_m = 1.5

h_amb = PropsSI("H", "T", T_amb + 273.15, "P", p_amb * 1e5, fluid)
s_amb = PropsSI("S", "T", T_amb + 273.15, "P", p_amb * 1e5, fluid)

p_1 = p_amb
T_1 = T_amb

p_2 = 10

h_1 = h_amb
s_1 = s_amb
s_2 = s_1

h_2 = PropsSI("H", "S", s_1, "P", p_2 * 1e5, fluid)

dot_W = dot_m * (h_2 - h_1)

dot_W
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mCoolProp[39;00m[38;5;21;01m.[39;00m[38;5;21;01mCoolProp[39;00m [38;5;28;01mimport[39;00m PropsSI
[1;32m      4[0m T_amb [38;5;241m=[39m [38;5;241m20[39m  [38;5;66;03m# Â°C[39;00m
[1;32m      5[0m p_amb [38;5;241m=[39m [38;5;241m1.01325[39m  [38;5;66;03m# bar[39;00m

[0;31mModuleNotFoundError[0m: No module named 'CoolProp'
ModuleNotFoundError: No module named 'CoolProp'

